PhpStorm Protocol (Win)/run_editor.js

```diff
@@ -6,7 +6,7 @@ var settings = {
     x64: true,
 
     // Set to folder name, where PhpStorm was installed to (e.g. 'PhpStorm')
-    folder_name: 'PhpStorm 2017.1.4',
+    folder_name: '<PhpStorm x.y.z folder name where the phpstorm executable has been installed>',
 
     // Set to window title (only text after dash sign), that you see, when switching to running PhpStorm instance
     window_title: 'PhpStorm 2017.1.4',
```

Comment:
Shouldn't this line match to one in README.md?

Reply:
Yeah, that makes sense. I was really attempting to highlight that it needs to be changed and point to the fact you need to find the folder that is holding the executable.

Reply:
updated